### DISCUSSION TITLE
What are the best UI options for this plugin?

### DISCUSSION THREAD

**Discussion by ewwwgiddings** [capawesome-team/capacitor-firebase#734] [Q&A]
There exists the `firebaseui` but what I have seen from that is the methods are tied to the actions of the `Firebase JS SDK`.

I am looking to use this plugin but I am wondering the best avenue for creating a compliant UI for all the social login buttons.

What is everyone using in order to get a compliant UI for all the social login buttons?

**Comment by robingenz** [comment]
Yes, the FirebaseUI is not available in this plugin. Usually everyone builds their own UI. You can get inspiration from sites like https://mobbin.com/. There is also an [Ionic Starter Template](https://ionstarter.dev/products/angular-firebase-starter) with a ready-made login screen.
